content/blog/kubernetes-monitoring-using-prometheus-and-grafana-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,419 @@
+          + namespace                  = "monitoring"
+          + pass_credentials           = false
+          + recreate_pods              = false
+          + render_subchart_notes      = true
+          + replace                    = false
+          + repository                 = "https://prometheus-community.github.io/helm-charts"
+          + reset_values               = false
+          + reuse_values               = false
+          + skip_crds                  = false
+          + status                     = "deployed"
+          + timeout                    = 300
+          + verify                     = false
+          + version                    = "23.0.0"
+          + wait                       = true
+          + wait_for_jobs              = false
+    
+          + set {
+              + name  = "prometheus-node-exporter.hostNetwork"
+              + value = "false"
+            }
+          + set {
+              + name  = "prometheus-node-exporter.hostPID"
+              + value = "false"
+            }
+          + set {
+              + name  = "prometheus-node-exporter.hostRootFsMount.enabled"
+              + value = "false"
+            }
+          + set {
+              + name  = "server.service.type"
+              + value = "NodePort"
+            }
+        }
+    
+    Plan: 2 to add, 0 to change, 0 to destroy.
+    
+    Do you want to perform these actions?
+      Terraform will perform the actions described above.
+      Only 'yes' will be accepted to approve.
+    
+      Enter a value: yes
+    
+    helm_release.grafana-dashboard: Creating...
+    helm_release.prometheus-stack: Creating...
+    helm_release.grafana-dashboard: Still creating... [10s elapsed]
+    helm_release.prometheus-stack: Still creating... [10s elapsed]
+    helm_release.grafana-dashboard: Still creating... [20s elapsed]
+    helm_release.prometheus-stack: Still creating... [20s elapsed]
+    helm_release.grafana-dashboard: Still creating... [30s elapsed]
+    helm_release.prometheus-stack: Still creating... [30s elapsed]
+    helm_release.grafana-dashboard: Still creating... [40s elapsed]
+    helm_release.grafana-dashboard: Creation complete after 43s [id=grafana-dashboard]
+    helm_release.prometheus-stack: Still creating... [40s elapsed]
+    helm_release.prometheus-stack: Still creating... [50s elapsed]
+    helm_release.prometheus-stack: Still creating... [1m0s elapsed]
+    helm_release.prometheus-stack: Still creating... [1m10s elapsed]
+    helm_release.prometheus-stack: Creation complete after 1m18s [id=prometheus-stack]
+    
+    Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
+```
+
+#### Check Prometheus and Grafana
+
+A﻿fter few minutes Terraform run, both Prometheus and Grafana will be deployed in the K8s cluster to the namespace *monitoring*. 
+
+T﻿yping the following command to check the deployed monitoring resources. All the Pods should be in *Running* states. 
```

Comment:
Change "Typing" to "Type"

You might want to rephrase the second sentence to "All the Pods should show that they are in Running states."